### ISSUE TITLE
Gin 2.1 and Java6 results in: Unsupported major.minor version 51.0 Error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [anne/google-gin#190]
```
It seems, that the current GIN 2.1 is compiled with Java7.

We still use Java6 for our development so we get the "Unsupported major.minor 
version 51.0" Error.


Is there an actual reason that Java7 is now required for Gin 2.1? Or is it 
possible to get something like 2.1.1 that works again with Java6?


Currently we are not ready to switch to Java7 but would like to have these 
changeset http://code.google.com/p/google-gin/source/detail?r=261 available 
(without the need to have our own compiled version  of Gin)


Thanks in advance :)
```

Original issue reported on code.google.com by `kujans...@gmail.com` on 11 Jul 2013 at 2:25

**Comment by GoogleCodeExporter** [comment]
```
We need to make a new release. Also it would be good to include the patch for 
issue 186 with the new release.
```

Original comment by `gok...@google.com` on 11 Jul 2013 at 11:29
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This doesn't really require a new release, just recompiled jars. A new minor 
version with 186 patched would be nice though - if someone has the time. :)
```

Original comment by `aragos` on 12 Jul 2013 at 1:15

**Comment by GoogleCodeExporter** [comment]
```
@aragos: in Maven land, this requires a new "release" (or a new "version" if 
you prefer; call it 2.1.0-1 if you like to make it clearer that this is a 
"packaging fix" for the 2.1.0 "logical" release). Artifacts are deemed 
immutable, so that if someone tells you "I use version 2.1.0" you can be sure 
it's the exact same as the 2.1.0 you're using, independently of the time you 
downloaded it.
```

Original comment by `t.broyer` on 12 Jul 2013 at 9:39

**Comment by GoogleCodeExporter** [comment]
```
Any ETA when it would be ready? :)
```

Original comment by `kujans...@gmail.com` on 23 Jul 2013 at 6:34

**Comment by GoogleCodeExporter** [comment]
```
Hey guys,

i don't want to bug you but i would like to know when 2.1.1 will be published 
to maven?

I saw there were some changes 2 weeks ago that look like 2.1.1 is "ready".

http://code.google.com/p/google-gin/source/detail?r=273 created a 2.1.1 tag
and
http://code.google.com/p/google-gin/source/detail?r=274 updated build.xml to 
produce the correct output jars (fixes this issue in the second of a 
build+release)


thanks in advance 

PS: a release would allow me to fix some TODOs in my code, thats why i ask :)
```

Original comment by `kujans...@gmail.com` on 27 Aug 2013 at 3:01

**Comment by GoogleCodeExporter** [comment]
```
Sorry for the delay, there were some issues with our maven repository. This has 
been fixed now and Gin 2.1.1 can be found on Maven Central: 
http://repo2.maven.org/maven2/com/google/gwt/inject/gin/2.1.1/
```

Original comment by `aragos` on 11 Sep 2013 at 6:21
- Changed state: **Fixed**
